Signal/src/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -140,10 +147,11 @@ class AttachmentApprovalViewController: UIViewController, OWSAudioAttachmentPlay
         audioPlayButton.autoSetDimension(.height, toSize:buttonSize)
         subviews.append(audioPlayButton)
 
-        if let fileExtensionLabel = createFileExtensionLabel() {
+        if let fileNameLabel = createFileNameLabel() {
+            subviews.append(fileNameLabel)
+        } else if let fileExtensionLabel = createFileExtensionLabel() {
             subviews.append(fileExtensionLabel)
         }
```

Comment:
oh right. I was thinking this was the message view, but it's the full screen approval view. Sounds right!